ams/server/src/main/java/com/netease/arctic/server/dashboard/controller/SettingController.java

```diff
@@ -58,19 +58,6 @@ public SettingController(Configurations serviceConfig, OptimizerManager optimize
    */
   public void getSystemSetting(Context ctx) {
     Map<String, String> config = new HashMap<>();
-    //Show core configuration first
-    putSetting(config, ArcticManagementConf.SERVER_EXPOSE_HOST.key(),
-        serviceConfig.get(ArcticManagementConf.SERVER_EXPOSE_HOST));
-    putSetting(config, ArcticManagementConf.THRIFT_BIND_PORT.key(),
-        serviceConfig.get(ArcticManagementConf.THRIFT_BIND_PORT));
-    putSetting(config, ArcticManagementConf.HTTP_SERVER_PORT.key(),
-        serviceConfig.get(ArcticManagementConf.HTTP_SERVER_PORT));
-    putSetting(config, ArcticManagementConf.DB_TYPE.key(), serviceConfig.get(ArcticManagementConf.DB_TYPE));
-    putSetting(config, ArcticManagementConf.DB_CONNECTION_URL.key(),
-        serviceConfig.get(ArcticManagementConf.DB_CONNECTION_URL));
-    putSetting(config, ArcticManagementConf.DB_CONNECTION_URL.key(),
-        serviceConfig.get(ArcticManagementConf.DB_CONNECTION_URL));
-
     serviceConfig.toMap().forEach((k, v) -> {
       if (!config.containsKey(k)) {
```

Comment:
It doesn't seem necessary to put the configuration into the `config` variable again here. Instead, we can directly sort the configuration and put it into the "result" variable below.

Reply:
will fix